ras_party/views/account_view.py

```diff
@@ -113,6 +113,21 @@ def change_respondent_enrolment_status():
     return make_response(jsonify('OK'), 200)
 
 
+@account_view.route('/respondents/disable-user-enrolments', methods=['PATCH'])
```

Comment:
I think `PATCH` is technically correct but I'm a little worried that it might cause people some issues because no-one ever uses it :D